css/main.css

```diff
@@ -27,7 +27,12 @@ header {
   color: white;
 }
```

Comment:
- [ ] Please consider using global CSS variables. They are values that can be reused throughout a stylesheet. They are similar to variables in programming languages, and they allow you to store and reuse values in your CSS code. You can read more about them [here](https://www.w3schools.com/css/css3_variables.asp)
It can be used like this 
```
:root {
	--global--color-primary: #28303d;
}

a {
	color: var(--global--color-primary);
}

.social-icons a {
	background: var(--global--color-primary);
}
```